nvidiaforceware.txt

```diff
@@ -0,0 +1,9 @@
+[Section]

```

Comment:
Please ensure that your file is encoded in UTF-8 without BOM, with CR-LF endings.

Reply:
CI is passing.